Makefile

```diff
@@ -57,10 +57,36 @@ build.examples.tinygo: $(tinygo_sources)
 c_sources := $(wildcard examples/*/testdata/*.c examples/*/*/testdata/*.c examples/*/testdata/*/*.c)
 .PHONY: build.examples.zig-cc
 build.examples.zig-cc: $(c_sources)
-	for f in $^; do \
+	@for f in $^; do \
 	    zig cc --target=wasm32-wasi -O3 -o $$(echo $$f | sed -e 's/\.c/\.wasm/') $$f; \
 	done
 
+# Here are the emcc args we use:
+#
+# * `-Oz` - most optimization for code size.
+# * `--profiling` - adds the name section.
+# * `-s STANDALONE_WASM` - ensures wasm is built for a non-js runtime.
+# * `-s EXPORTED_FUNCTIONS=_malloc,_free` - export allocation functions so that
```

Comment:
this and below aren't needed for our current example, but are needed for things like imports. Since they take a while to figure out, I left them in.

Ex. this would require the WARM_ON_UNDEFINED_SYMBOLS=0 thing:
```c++
/* _log is a WebAssembly import which prints a string (linear memory offset,
byteCount) to the console. */
EM_IMPORT(log) extern "C" void _log(const char* ptr, uint32_t size);
```